pkg/arvo/sys/hoon.hoon

```diff
@@ -7198,7 +7200,6 @@
   |=  [p=(pair type note) q=type]
   ^-  type
   ?:  =(%void q)  %void
-  ?:  =(%noun q)  %noun
```

Comment:
leaving myself a reminder to try and remember why I did this and whether I actually still need it